.flake8

```diff
@@ -0,0 +1,26 @@
+[flake8]
+exclude =
+    ./tools/*,
+    ./venv/*,
+    ./twiml/*,
+    */testable_snippets/*
+
+ignore =
+    # D100: Missing docstring in public module
+    D100,
+    # H101: Use TODO(NAME)
+    H101,
+    # H202: assertRaises Exception too broad
+    H202,
+    # H306: imports not in alphabetical order (time, os)
+    H306,
+    # H401: docstring should not start with a space
+    H401,
+    # H403: multi line docstrings should end on a new line
+    H403,
+    # H404: multi line docstring should start without a leading new line
+    H404,
+    # H405: multi line docstring summary not separated with an empty line
+    H405,
+    # H501: Do not use self.__dict__ for string formatting
+    H501
```

Comment:
I think that some of those shouldn't be ignored but fixed if possible. Namely H101 H202 H40x H501